trulens_eval/trulens_eval/tests/groundedness_benchmark_snowflake_arctic.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>if easy, can you show turbo in the same leaderboard as the others?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Would be great to keep the NLI results here as well

Reply:
<p>+1 if the command is same <code>get_leaderboard</code>, why does it get two different outputs ?</p>

Reply:
@sfc-gh-pdharmana there's a bug at the moment that Piotr is working on which prevents us to initialize TruBasicApp in a row. 
Basically defining more than one TruBasicApp in a row and calling recording with context does not work - only the recorder defined last in the order will trigger adding new records to DB. Refer to this [notebook](https://github.com/truera/trulens/blob/7c89f53a7fb15836e1391dfebe804c4f32011bec/trulens_eval/examples/quickstart/text2text_quickstart.ipynb) for a minimal reproducible example